.github/workflows/release_workflow.yml

```
@@ -61,7 +61,7 @@ jobs:
           git commit -m "Update documentation" -a || true
 
       - name: Push Changes
-        uses: ad-m/github-push-action@master
+        uses: ad-m/github-push-action@main
```

```suggestion
        uses: ad-m/github-push-action@master
```
The branch reference here shouldn't be updated since it is for another repo.